poup.css

```diff
@@ -0,0 +1,233 @@
+.popup_window {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  display: none;
+  top: 1%;
+  bottom: 0;
+  overflow: auto;
+}
+
+.popup_skills_mob li {
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+  background-color: #ebecf0;
+  color: #344563;
+  padding: 10px 10px;
+  border-radius: 5px;
+  margin-right: 1%;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 13px;
+  justify-content: center;
+  width: 80%;
+  padding: 75px 0;
+  margin-left: 8%;
+  background-color: #fff;
+}
+
+.popup .mob_snap {
+  width: 90%;
+  text-align: center;
+  align-items: center;
+
+  /* display: flex;
+   height: 220px;
+  padding: 8px 8px 172px 263px;
+  justify-content: flex-end;
+  align-items: center;
+  border-radius: 8px; */
```

Comment:
- [ ] Please 🙏🏻 remove these line of code from your `file` because having commented code in the codebase makes it cluttered, confuses developers, and can create maintenance and version control issues. Kindly fix it and it should be avoided for clean, professional and standardized code.